docs/resources/environment.md

```diff
@@ -50,6 +51,9 @@ resource "axual_environment" "development" {
   authorization_issuer = "Auto"
   instance = data.axual_instance.testInstance.id
   owners = axual_group.tenant_admin_group.id
+   settings = {
```

Comment:
I can't update my terraform resource to delete the settings in this way, because Terraform thinks that nothing has changed. If I change to `settings={}`, it works. Please cover this change that doesn't work currently with a test:
ORIGINAL env:
```
resource "axual_environment" "tf-test-env" {
  name = "tf-development"
  short_name = "tfdev"
  description = "This is the terraform testing environment1"
  color = "#19b9be"
  visibility = "Private"
  authorization_issuer = "Auto"
  instance = "ee6e12e5301b41bf8a00ef3388806f17"
  owners = axual_group.team-integrations.id
   settings ={
     enforceDataMasking = "true",
     testKey = "TestValue1"
   }
}
```
UPDATED env:
```
resource "axual_environment" "tf-test-env" {
  name = "tf-development"
  short_name = "tfdev"
  description = "This is the terraform testing environment1"
  color = "#19b9be"
  visibility = "Private"
  authorization_issuer = "Auto"
  instance = "ee6e12e5301b41bf8a00ef3388806f17"
  owners = axual_group.team-integrations.id
}
```

Reply:
In the API, if you do not provide the settings as part of the request, it means there is no change. if you set it to the null or empty map, it's equal to removing the entire content of the map.